audit/audit-gcp.sh

```diff
@@ -35,17 +47,18 @@ gcloud projects list \
     for service in `gcloud services list --filter state:ENABLED --format=json \
                     | jq -r .[].config.name | sed s:.googleapis.com::`
     do
+        echo "##### Processing: $service #####"
         case $service in
             compute)
-                echo TODO: $service Needs compute.projects.get
-                #### gcloud compute project-info describe
-                #### gcloud compute instances list --format=$format > $PROJECT/services/compute.instances.$format
-                #### gcloud compute disks list --format=$format > $PROJECT/services/compute.disks.$format
+                #echo TODO: $service Needs compute.projects.get
+                # Only record usages so that quota increases/decreases do not alter the audit
+                gcloud compute project-info describe --format="json(quotas)" | jq ".quotas[] |= del(.limit)" > $PROJECT/services/compute.resources.json
+                gcloud compute instances list --format=$format > $PROJECT/services/compute.instances.$format
```

Comment:
I don't think we should do this - it will change, for example as clusters are upgraded

Reply:
Makes sense, any autoscaling would also trigger a difference in the audit files. Removed.